src/applications/vaos/appointment-list/sass/print.scss

```diff
@@ -10,13 +10,14 @@
   button,
   .menu,
   .hero,
-  .adslot {
+  [data-widget-type="banner"],
+  [data-widget-type="maintenance-banner"] {
```

Comment:
This is very interesting. Looking at the staging, there appears a total of 4 data-widget-type elements. While these "banners" are not visible now, it may be on the safe side to add them on the hidden print list

data-widget-type="header"
data-widget-type="banner"
data-widget-type="maintenance-banner"
data-widget-type="promo-banner"

Reply:
Yea, just saw the promo-banner type. There doesn't seem to be a common list of banners that is update each time a new banner is created (or I can't find it). This means our .scss file will have to be updated each time a new banner type is created. Maybe I'll asked the question in Slack.

Reply:
This is the first i've come across `data-widget-type`. And it looks like they are using it as an attribute to an element. Hopefully there's a pre-defined set of banner names.